manifests/qpid_client.pp

```diff
@@ -0,0 +1,44 @@
+# Pulp Master Certs configuration
+class certs::qpid_client (
+
+  $hostname   = $::certs::node_fqdn,
+  $generate   = $::certs::generate,
+  $regenerate = $::certs::regenerate,
+  $deploy     = $::certs::deploy,
+
+  $nodes_cert_dir   = $certs::params::nodes_cert_dir,
+  $nodes_cert_name  = $certs::params::nodes_cert_name,
+
+  $messaging_ca_cert     = $certs::ca_cert,
+  $messaging_client_cert = $certs::params::messaging_client_cert
+
+  ) inherits pulp::params { # lint:ignore:inherits_across_namespaces
+
+  cert { "${::certs::qpid_client::hostname}-qpid-client-cert":
+    hostname      => $hostname,
+    common_name   => 'pulp-qpid-client-cert',
+    purpose       => client,
+    country       => $::certs::country,
+    state         => $::certs::state,
+    city          => $::certs::city,
+    org           => 'PULP',
+    org_unit      => $::certs::org_unit,
+    expiration    => $::certs::expiration,
+    ca            => $::certs::default_ca,
+    generate      => $generate,
+    regenerate    => $regenerate,
+    deploy        => $deploy,
+    password_file => $certs::ca_key_password_file,
+  }
+
+  if $deploy {
+
+    file { $messaging_client_cert:
+      owner => 'apache',
+      group => 'apache',
+      mode  => '0640',
+    } -> Class['pulp::config']
```

Comment:
You also need this part https://github.com/Katello/puppet-certs/pull/78/files#diff-17ce3af01e46365ea61a9b96d0f63574L69 as that is what sets up the cert